src/modules/commander/commander_params.c

```diff
@@ -87,58 +87,86 @@ PARAM_DEFINE_FLOAT(TRIM_PITCH, 0.0f);
 PARAM_DEFINE_FLOAT(TRIM_YAW, 0.0f);
 
 /**
- * Empty cell voltage.
+ * Number of battery cells.
  *
- * Defines the voltage where a single cell of the battery is considered empty.
+ * Defines the number of cells the attached battery consists of.
  *
  * @group Battery Calibration
- * @unit V
+ * @unit S
+ * @min 1
+ * @max 10
  */
-PARAM_DEFINE_FLOAT(BAT_V_EMPTY, 3.4f);
+PARAM_DEFINE_INT32(BAT_N_CELLS, 1);
 
 /**
- * Full cell voltage.
+ * Full battery cell voltage.
  *
- * Defines the voltage where a single cell of the battery is considered full.
+ * Defines the voltage at which one battery cell is considered full.
  *
  * @group Battery Calibration
  * @unit V
+ * @min 0.0
  */
-PARAM_DEFINE_FLOAT(BAT_V_CHARGED, 4.2f);
+PARAM_DEFINE_FLOAT(BAT_V_FULL, 0.0f);
 
 /**
- * Voltage drop per cell on 100% load
+ * Low battery cell voltage.
  *
- * This implicitely defines the internal resistance
- * to maximum current ratio and assumes linearity.
+ * Defines the voltage at which one battery cell is considered low.
  *
  * @group Battery Calibration
  * @unit V
- * @min 0.0f
+ * @min 0.0
  */
-PARAM_DEFINE_FLOAT(BAT_V_LOAD_DROP, 0.07f);
+PARAM_DEFINE_FLOAT(BAT_V_LOW, 0.0f);
 
 /**
- * Number of cells.
+ * Critical battery cell voltage.
  *
- * Defines the number of cells the attached battery consists of.
+ * Defines the voltage at which one battery cell is considered critically empty.
  *
  * @group Battery Calibration
- * @unit S
- * @min 1
- * @max 10
+ * @unit V
+ * @min 0.0
  */
-PARAM_DEFINE_INT32(BAT_N_CELLS, 3);
+PARAM_DEFINE_FLOAT(BAT_V_CRIT, 0.0f);
 
 /**
  * Battery capacity.
  *
  * Defines the capacity of the attached battery.
+ * Setting this value to -1 disables remaining battery estimation based on discharged capacity.
  *
  * @group Battery Calibration
- * @unit mA
+ * @unit mAh
  */
-PARAM_DEFINE_FLOAT(BAT_CAPACITY, -1.0f);
+PARAM_DEFINE_INT32(BAT_CAPACITY, -1);
```

Comment:
What is the difference between the two? Seems both cases would bypass discharged capacity.